.eslintrc.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
     '@typescript-eslint/indent': ['error', 2],
     '@typescript-eslint/explicit-function-return-type': 'off',
     '@typescript-eslint/explicit-member-accessibility': 'off',
+    '@typescript-eslint/no-var-requires': 'off',
```

Comment:
Why do we need this one?

Reply:
I think that was something wrong I was doing when I was linking and working. The compiler threw an error for require. I fixed it now.